platform-tests/src/platform/acceptance/plain_http_test.go

```diff
@@ -21,10 +21,14 @@ var _ = Describe("plain HTTP requests", func() {
 
 	Describe("to the API", func() {
 		It("has the connection refused", func() {
-			uri := testConfig.ApiEndpoint + ":80"
-			_, err := net.DialTimeout("tcp", uri, CONNECTION_TIMEOUT)
-			Expect(err).To(HaveOccurred(), "should not connect")
-			Expect(err.(net.Error).Timeout()).To(BeTrue(), "should timeout")
+			req, err := http.NewRequest("GET", fmt.Sprintf("http://%s/v2/info", testConfig.ApiEndpoint), nil)
+			Expect(err).NotTo(HaveOccurred())
+			_, err = http.DefaultTransport.RoundTrip(req)
+			Expect(err).To(HaveOccurred())
+			Expect(err).To(MatchError(Or(
+				ContainSubstring("connection refused"),
+				ContainSubstring("connection reset by peer"),
+			)))
```

Comment:
This relates to weirdness that @henrytk and I found. ELBs configured to not listen on port 80 still establish TCP connections on port 80, under some conditions. We've been talking to AWS support about it, and they seemed surprised themselves. Search your email inbox for `TCP connections to ports not covered by security groups` to read that AWS support thread.

Concourse gets `connection refused`. Running from our own computers gets `connection reset by peer`. I decided to allow both, as the connection being established but then immediately closed (`connection reset by peer`) is the behaviour that outside tenants will see.

Reply:
It'll be the transparent proxy in our office that's accepting the connections 🐼

Reply:
🤦‍♀️

Reply:
@46bit do you mind if I update the commit message with the nugget of @dcarley wisdom?

Reply:
I've taken an EXECUTIVE DECISION and done it.